vm-setup/roles/packages_installation/tasks/centos_required_packages.yml

```diff
@@ -0,0 +1,53 @@
+# Perform CentOS/RHEL8 related configurations
+- name: Fail if CONTAINER_RUNTIME is not set to podman
+  fail:
+    msg: Only Podman is supported in CentOS/RHEL8
+  when: CONTAINER_RUNTIME != "podman"
+
+- name: Enable SELinux
+  ansible.posix.selinux:
+    policy: targeted
+    state: permissive
+  when: ansible_selinux.status == "enabled"
+
+# Remove any previous tripleo-repos to avoid version conflicts
+# (see FIXME re oniguruma below)
+- name: Remove any previous tripleo-repos to avoid version conflicts
+  dnf:
+    name: python*-tripleo-repos
+    state: absent
+
+- name: Upgrade all packages
+  dnf:
+    name: "*"
+    state: latest
+
+# Install additional repos as needed for each OS version
+- name: Enable external repository
+  dnf:
+    name: extras
```

Comment:
```suggestion
    name: epel-release
```

Reply:
Thanks @furkatgofurov7 . Will try that!

Reply:
Note that epel isn't the same as extras, and there is some package overlap with different versions ref http://mirror.centos.org/centos/8/extras/x86_64/os/Packages/

I wonder if we need this repo at all, since the main dependency there seems to be ansible, which we're now [installing via pip](https://github.com/metal3-io/metal3-dev-env/blob/master/01_prepare_host.sh#L30)?

Reply:
Good catch. Removed it.